### PR TITLE
k8s: service metadata

### PR DESCRIPTION
**Pull request by hunchback**
To test run the below and verify using UI:

```
kubectl create -f tests/k8s/statefulset.yaml
```

### CODE REVIEW

**Comment by hunchback** [comment]
run tests

**Comment by hunchback** [comment]
run tests

File: topology/probes/k8s/service.go [hunk 1]
```diff
@@ -40,11 +40,18 @@ type serviceProbe struct {
...
 
 func dumpService(srv *v1.Service) string {
-	return fmt.Sprintf("service{Name: %s}", srv.GetName())
+	return fmt.Sprintf("service{Namespace: %s, Name: %s}", srv.Namespace, srv.Name)
 }
 
 func (p *serviceProbe) newMetadata(srv *v1.Service) graph.Metadata {
-	return newMetadata("service", srv.Namespace, srv.GetName(), srv)
+	m := newMetadata("service", srv.Namespace, srv.Name, srv)
+	m.SetFieldAndNormalize("Ports", s.Spec.Ports)
```
Comment (lebauce) [inline topology/probes/k8s/service.go]: Did you mean `srv.Spec.Ports` instead of `s.Spec.Ports` (s is undefined)

Comment (hunchback) [reply topology/probes/k8s/service.go]: yes, fixed!

**Comment by hunchback** [comment]
kindly review/merge, thanks

**Pull request merged**